vars/tagDeployment.groovy

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env groovy
 
-def call(String microservice, String aws_profile = "test", String tag = null) {
-  tag = tag ?: gitCommit()
-
-  build job: 'run-tag-and-capture-notes-commit-based',
```

Comment:
where is the current code for this? i can't find it :(

Reply:
Linked to in the PR description?